dist/index.js

```diff
@@ -14022,6 +14022,7 @@ async function exportSecrets() {
     const vaultNamespace = core.getInput('namespace', { required: false });
     const extraHeaders = parseHeadersInput('extraHeaders', { required: false });
     const exportEnv = core.getInput('exportEnv', { required: false }) != 'false';
+    const exportToken = core.getInput('exportToken', { required: false }) == 'false';
```

Comment:
For quick clarification: does `exportToken: false` _enable_ exporting the token? Because that's what's currently hapenning here?

Reply:
I fix it when I write unit tests ;)